lib/human_player.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './player.rb'
+
+class HumanPlayer < Player
```

Comment:
Code Smell: Refused Bequest/Speculative Generality.  Same comment as with computer player - this subclass inherits nothing from parent class.  Other features were perhaps once considered, but never implemented.  If inheritance is not needed, deleting superclass would make the code cleaner.